.github/workflows/coverage-report.yml

```diff
@@ -0,0 +1,59 @@
+name: Regression Tests Coverage Report
+
+on:
+  pull_request_target:
+    paths:
+    - "examples/**"
+    - "test/**"
```

Comment:
We should filter out examples/landmarks -- `"- !examples/landmarks/**"`

Reply:
Good call! Done